client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -373,30 +373,50 @@ export default React.createClass( {
 
 	unverifiedDomainsCanManage() {
 		const domains = this.getDomains().filter( domain => domain.isPendingIcannVerification && domain.currentUserCanManage );
+		const severity = domains.some( ( { registrationMoment } ) =>
```

Comment:
I think we should break this into two variables and add some more parenthesis and curly braces here for more readability. Something like:
```js
const isWithinTwoDays = domains.some( ( { registrationMoment } ) => {
    if ( registrationMoment ) {
       return false;
    }
    return moment( registrationMoment ).add( 2, 'days' ).isAfter( moment() ) );
} );
 const severity = isWithinTwoDays ? 'is-info' : 'is-error';
```

Warning: I didn't test the above code as I just tried to figure out what your code was doing and match it up